### PR TITLE
docs(preset): new preset "Pastel Rainbow"

### PR DESCRIPTION
**Pull request by ecspresso**
#### Description
Created a new preset named "Pastel Rainbow". It is the same as Gruvbox Rainbow but with the colors of Pastel Powerline.

#### Motivation and Context

Because I preferred the colors of Pastel Powerline but liked the formating of Gruvbox. I wanted to share my version.

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [X] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [X] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

**Comment by ecspresso** [comment]
Please help me correct any errors. I could not find any other documentation for adding a new preset other than "edit this readme".

**Comment by ecspresso** [comment]
I can see that there is an error but I cannot tell what.

**Comment by jemhv** [comment]
> I can see that there is an error but I cannot tell what.

well in the docs/presets/readme.md file you have two line gap before  ## [Pastel Rainbow](./pastel-rainbow.md), please remove one gap and you're good to go.

File: docs/presets/README.md [hunk 1]
```diff
@@ -80,3 +80,10 @@ This preset is heavily inspired by [Pastel Powerline](./pastel-powerline.md), an
 This is a pseudo minimalist preset inspired by the [geometry](https://github.com/geometry-zsh/geometry) and [spaceship](https://github.com/spaceship-prompt/spaceship-prompt) prompts.
 
 [![Screenshot of Jetpack preset](/presets/img/jetpack.png "Click to view Jetpack preset")](./jetpack)
+
```
Comment (jemhv) [inline docs/presets/README.md:83]: remove this line please

**Comment by ecspresso** [comment]
Found one more lined that was causing an error, removed that too.

The last error seems to be my reference to gruvbox-rainbow.toml which is not in my PR but is the one I based it on.

Do I need to fix that?

File: docs/presets/pastel-rainbow.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[Return to Presets](./README.md#gruvbox-rainbow)
```
Comment (jemhv) [inline docs/presets/pastel-rainbow.md:1]: this has to be your file path which is (./#pastel-rainbow)

File: docs/presets/pastel-rainbow.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+### Configuration
+
+```sh
+starship preset pastel-rainbow -o ~/.config/starship.toml
+```
+
+[Click to download TOML](/presets/toml/pastel-rainbow.toml)
+
+<<< @/.vuepress/public/presets/toml/gruvbox-rainbow.toml
```
Comment (jemhv) [inline docs/presets/pastel-rainbow.md:21]: change the path <<< @/public/presets/toml/pastel-rainbow.toml

File: docs/presets/pastel-rainbow.md (hunk 2 above)
Comment (ecspresso) [reply docs/presets/pastel-rainbow.md]: lol thanks, didn't see that.

File: docs/presets/pastel-rainbow.md (hunk 3 above)
Comment (ecspresso) [reply docs/presets/pastel-rainbow.md]: Thanks, I have edited both errors.

**Comment by jemhv** [comment]
Why did you put your file in .vuepress? it should be in presets that's right in docs.

File: docs/.vuepress/public/presets/img/pastel-rainbow.png
Comment (jemhv) [inline docs/.vuepress/public/presets/img/pastel-rainbow.png:1]: put the img and toml files inside the right directory, docs/public/presets/

Comment (davidkna) [reply docs/.vuepress/public/presets/img/pastel-rainbow.png:1]: The paths changed when we switched to vitepress.

**Comment by davidkna** [comment]
Instead of a new theme, I would prefer adding a new `palettes` definition to `Gruvbox Rainbow` instead.

**Comment by ecspresso** [comment]
Thanks for all the help, I'll close this for now and might take it back up when the kids are older I have more time.

**Pull request closed without merging**